app/scripts/controllers/newOverview.js

```diff
@@ -301,11 +301,15 @@ function OverviewController($scope,
   };
 
   var filterByLabel = function(items) {
-    return LabelFilter.getLabelSelector().select(items);
+    var labelSelector = LabelFilter.getLabelSelector();
+    if (labelSelector.isEmpty()) {
+      return items;
+    }
```

Comment:
@jwforres The label `LabelSelector.select()` rejects objects with no labels even when there are no active filters. Is this a label selector bug?